### PR TITLE
Tbh 186 load translations for tb health check

### PR DESCRIPTION
**Pull request by Buhle79**

### CODE REVIEW

File: base/actions/actions.py [hunk 1]
```diff
@@ -696,6 +695,10 @@ async def submit(
                         resp = await client.post(url, json=post_data, headers=headers)
                         # TODO: remove print
                         print(resp.content)
+
+                        template, group_arm = utils.get_display_message_template(resp)
+                        self.SLOTS.append(group_arm)
```
Comment (bearnard) [inline base/actions/actions.py]: why is this needed?

Comment (Buhle79) [reply base/actions/actions.py]: No we don't need it, i was testing with it.

File: base/actions/actions.py [hunk 2]
```diff
@@ -704,14 +706,171 @@ async def submit(
...
+        return []
+
+
+class GroupArmForm(BaseFormAction):
+    YES_NO_MAPPING = {
+        "yes": True,
+        "no": False,
+    }
+
+    GENDER_MAPPING = {
```
Comment (bearnard) [inline base/actions/actions.py]: why do you need to define all these class properties in this class too?

Comment (Buhle79) [reply base/actions/actions.py]: I use them to map user response options, as they are required fields on api

File: base/data/stories.md [hunk 3]
```diff
@@ -25,10 +28,13 @@
...
     - form{"name": "tbcheck_form"}
     - form{"name": null}
+    - group_arm_form
+    - form{"name": "group_arm_form"}
+    - form{"name": null}
     - action_session_start
 
 ## do optin
-* opt_in
+ * opt_in
```
Comment (bearnard) [inline base/data/stories.md]: extra space?

**Comment by bearnard** [comment]
👍  LGTM
It may be worth trying to break out more stuff into functions, i.e all the httpx boilerplate, but welcome to do that at a later stage...

File: base/actions/actions.py (hunk 2 above)
Comment (Buhle79) [reply base/actions/actions.py]: No longer needed as i do patch request
